### PR TITLE
Add Ladle

### PR DESCRIPTION
**Pull request by tajo**
(works locally but playwright might not work in your CI, let's see how this PR goes)

### CODE REVIEW

**Comment by tajo** [comment]
Ah, this PR doesn't run our tests but I suspect that

```
pnpm playwright install chromium
```

needs to be added ([as we do](https://github.com/tajo/ladle/blob/master/.github/workflows/ci.yml#L38)), there is `beforeInstall`, maybe adding `postInstall` too?

Or I can create a new `playwright+test` command just for this.

**Comment by patak-dev** [comment]
Thanks for the PR!
Maybe a new command would work here. Svelte also uses playwright for its tests, maybe you could check how they are doing it? I think several other projects use it too. If an `afterInstall` command is useful though, no problem adding it too

**Comment by dominikg** [comment]
I'd prefer beforeBuild and beforeTest commands to be in sync with beforeInstall. But it should be possible to just do `test : "pnpm playwright install && pnpm test"`

**Comment by tajo** [comment]
Ok, should be good to go.

**Comment by dominikg** [comment]
beforeTest and beforeBuild work now, as well as passing shell scripts as string.

File: tests/ladle.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { runInRepo, $ } from '../utils'
+import { RunOptions } from '../types'
+
+export async function test(options: RunOptions) {
+	await runInRepo({
+		...options,
+		repo: 'tajo/ladle',
+		branch: 'master',
+		build: 'build',
+		test: [async () => $`pnpm playwright install chromium`, 'test']
```
Comment (dominikg) [inline tests/ladle.ts:10]:
```suggestion
    beforeTest: 'pnpm playwright install chromium'
		test: 'test'
```

**Pull request closed without merging**